src/client/pages/Download/components/DownloadHeadline.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { useTextKeys } from 'utils/textKeys'
+import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+
+const Headline = styled.div`
+  width: 100%;
+  margin-bottom: 3rem;
+  font-size: 2rem;
+  line-height: 40px;
+  text-align: center;
+  color: ${colorsV3.gray100};
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+    font-size: 3rem;
+    line-height: 56px;
+    text-align: left;
+  }
+`
+
+const MainHeadline = styled.h1`
+  margin: 0;
+  font-size: inherit;
+  display: inline;
+`
+
+export const DownloadHeadline: React.FC = () => {
+  const textKeys = useTextKeys()
+
+  return (
+    <Headline>
+      <MainHeadline>
+        {textKeys.ONBOARDING_DOWNLOAD_HEADLINE_PART_1() + ' '}
+      </MainHeadline>
+      <span>{textKeys.ONBOARDING_DOWNLOAD_HEADLINE_PART_2()}</span>
```

Comment:
Nitpick, but should this not be part of the h1?

Reply:
I did it this way to get the headline to line break in the right place in desktop but not line break at all in mobile. I thought other solutions to achieve this looked... weirder, so if it's not a deal breaker for you guys I'm going to keep it this way. 😀

Reply:
you mean like having a media query on span to display block/inline? 
It is absolute **_not_** a deal breaker for me 😺

Reply:
_Nevermind_  I made the `div` an `h1` and of course if worked without changing `display` of `span`s since it's a flex box 🤦‍♀️